docs/folder-edit.directive.md

```diff
@@ -0,0 +1,41 @@
+# Folder Edit directive
+
+<!-- markdown-toc start - Don't edit this section.  npm run toc to generate it-->
+
+<!-- toc -->
+
+- [Basic Usage](#basic-usage)
+  * [Properties](#properties)
+  * [Events](#events)
+- [Details](#details)
+
+<!-- tocstop -->
+
+<!-- markdown-toc end -->
+
+## Basic Usage
+
+```html
+<adf-toolbar title="toolbar example">
+    <button mat-icon-button
+            [disabled]="!documentList.selection || !documentList.selection.length || !documentList.selection[0].entry.isFolder"
```

Comment:
this example I thing is too specific this documentlist obj is not present here

Reply:
yeah, you're right, it is not necessary, but it would be good, though, to let the users know that there must be a selection made in order for the edit dialog to open. I added this detail in the description and removed the [disable] part.